apps/client-server/src/app/websites/website.ts

```diff
@@ -42,8 +44,10 @@ export abstract class Website<D extends DynamicObject> {
    * information for a website instance.
    *
    * Commons things that go here would be folders.
+   *
+   * This is not persisted across app restarts.
    */
-  protected readonly retrievedWebsiteData: DynamicObject = {};
+  protected readonly sessionData: DynamicObject = {};
```

Comment:
It kept popping up everytime i typed return, so i decided to rename it to more short and intutive name